apps/aeutil/lib/serialization.ex

```diff
@@ -554,7 +554,13 @@ defmodule Aeutil.Serialization do
     header_txs_hash_size = Application.get_env(:aecore, :bytes_size)[:txs_hash]
     header_root_hash_size = Application.get_env(:aecore, :bytes_size)[:root_hash]
     pow_evidence_size = Application.get_env(:aecore, :bytes_size)[:pow_total_size]
-    pow_to_binary = pow_to_binary(header.pow_evidence)
+
+    pow_to_binary =
+      if header.pow_evidence != :no_value do
+        pow_to_binary(Enum.reverse(header.pow_evidence))
```

Comment:
I think we should get rid of this Enum.reverse and just use Enum.sort over accumulator in pow_to_binary function.